src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -277,6 +294,16 @@ const TimeSeriesCard = ({
     [isEditable, series]
   );
 
+  // Set the colors for each dataset
+  const colors = { identifier: 'group', scale: {} };
```

Comment:
can we export and write a testcase to cover this function.  Is this sharable between BarChart and TimeSeries?

Reply:
As the props are different at the moment, I think it would be overly complex if it was shared. Once the props are refactored, they should be able to share the function

Reply:
Added testcase